### PR TITLE
Added API basics.

### PR DESCRIPTION
**Pull request by fpiagent**
This initial feature just defines the basics of the API, the README and the .gitignore.

### CODE REVIEW

File: assets/api/examples/event.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "eventType": "UPVOTE",
+    "playlistId": 2
```
Comment (leorod) [inline assets/api/examples/event.js]: I know this is just an example, but I want to make sure, are we using numeric IDs or UUIDs?

File: assets/api/schemas/event.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  "$schema": "http://json-schema.org/draft-03/schema",
+  "id": "http://jsonschema.net",
+  "required": true,
+  "properties": {
+    "eventType": {
+      "type": "number",
+      "required": true
+    },
+    "playlistId": {
+      "type": "number",
```
Comment (leorod) [inline assets/api/schemas/event.js]: Wouldn't it be better if we used UUIDs?

Comment (fpiagent) [reply assets/api/schemas/event.js]: uuid for everything? Well, after reading this I think we can give it a shot: https://blog.codinghorror.com/primary-keys-ids-versus-guids/
(I specially like the part where we don't depend on the DB to generate a new ID)

Comment (leorod) [reply assets/api/schemas/event.js]: Yep! They're bigger and all, but there's the DB independence + they're really collision-safe and easy to generate/work with

**Pull request merged**